Update README.md (#37)
percona/percona_pg_telemetry: README.md

  * Outputs the version of the extension.

2. `percona_pg_telemetry_status()`

  * OUT: latest_output_filename   - TEXT
    - Path of the JSON output file.
  * OUT: pt_enabled               - BOOLEAN
    - True if telemetry data collection is enabled.

> > **NOTE** Functions are only visible if extension has already been created using 'Create extension percona_pg_telemetry;'.

Reviewer: ```suggestion
> **NOTE** Functions are only visible after you have created the extension using the 'CREATE EXTENSION percona_pg_telemetry;' command.
```
